### PR TITLE
Postgre on GAE Standard for Java8

### PR DESCRIPTION
**Pull request by kurtisvg**
This is a replacement for @lesv's PR [here](https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/pull/52).

### CODE REVIEW

**Comment by kurtisvg** [comment]
@iangudger - PTAL

File: examples/postgres/appengine-standard-java8/pom.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    <dependency>
+      <groupId>com.google.cloud.sql</groupId>
+      <artifactId>postgres-socket-factory</artifactId>
+      <version>1.0.5-SNAPSHOT</version>
+    </dependency>
+    <!-- [END dependencies] -->
+  </dependencies>
+
+  <build>
+    <!-- for hot reload of the web application -->
```
Comment (iangudger) [inline examples/postgres/appengine-standard-java8/pom.xml]: Full sentence like above?

File: examples/postgres/appengine-standard-java8/src/main/java/com/example/appengine/postgresql/PostgreSqlServlet.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+public class PostgreSqlServlet extends HttpServlet {
+
+  Connection conn;
+
+  @Override
+  public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException,
+      ServletException {
+
+    final String createTableSql = "CREATE TABLE IF NOT EXISTS visits ( visit_id SERIAL NOT NULL, "
+        + "user_ip VARCHAR(46) NOT NULL, ts timestamp NOT NULL, "
```
Comment (iangudger) [inline examples/postgres/appengine-standard-java8/src/main/java/com/example/appengine/postgresql/PostgreSqlServlet.java]: My comments from before still apply. VARCHAR is absolutely the wrong type for storing IP addresses.

File: examples/postgres/appengine-standard-java8/src/main/java/com/example/appengine/postgresql/PostgreSqlServlet.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    PrintWriter out = resp.getWriter();
+    resp.setContentType("text/plain");
+
+    // store only the first two octets of a user's ip address
+    String userIp = req.getRemoteAddr();
+    InetAddress address = InetAddress.getByName(userIp);
+    if (address instanceof Inet6Address) {
+      // nest indexOf calls to find the second occurrence of a character in a string
+      // an alternative is to use Apache Commons Lang: StringUtils.ordinalIndexOf()
+      userIp = userIp.substring(0, userIp.indexOf(":", userIp.indexOf(":") + 1)) + ":*:*:*:*:*:*";
```
Comment (iangudger) [inline examples/postgres/appengine-standard-java8/src/main/java/com/example/appengine/postgresql/PostgreSqlServlet.java]: My comments on this from before still apply. This IPv6 parsing code is not correct.

**Pull request merged**